src/GovUk.Education.ExploreEducationStatistics.Data.Model/Database/StatisticsDbContext.cs

```diff
@@ -96,7 +96,7 @@ private void Configure(int? timeout = null, bool updateTimestamps = true)
                 ChangeTracker.Tracked += DbContextUtils.UpdateTimestamps;
             }
 
-            if (timeout.HasValue)
+            if (timeout.HasValue && Database.IsRelational())
```

Comment:
Yup! When the context is initialised through the test app, it hits this code path and throws an error as the option isn't available for the in-memory database. We need to this to prevent that error.